### DISCUSSION TITLE
There is no realization of `gx`. Is there a plan to achieve it?

### DISCUSSION THREAD

**Discussion by EXHades** [JetBrains/ideavim#322] [General]
https://youtrack.jetbrains.com/issue/VIM-1341

https://vimhelp.org/pi_netrw.txt.html#netrw-gx
https://stackoverflow.com/questions/9458294/open-url-under-cursor-in-vim-with-browser

**Comment by AlexPl292** [comment]
Hi, no immediate plans. However, this should be simple to implement, since IJ already recognizes links in the editor. We just need to find an API for that.

**Reply by EXHades** [reply]
Thank

**Reply by andrewgazelka** [reply]
Going to bump this discussion. I would love if someone could achieve it 👀. Couldn't we just call macOS `open`?

**Reply by citizenmatt** [reply]
While it's not traditional Vim, you can use `gd` on a link to navigate.

**Reply by andrewgazelka** [reply]
hmmm. maybe we could set a keybind `nmap gx gd` by default? makes it easier to discover...

**Reply by citizenmatt** [reply]
I think we could create a simple command that checks it's on a hyperlink and then invokes `GoToDefinition`. I don't think that would be too tricky. It's all about finding the time 😀
